jdk-wrapper.sh

```diff
@@ -99,15 +99,13 @@ safe_command "rm \"${l_fifo}\""
 command=
 cmd_configuration=
 for arg in "$@"; do
-  if [ -z ${in_command} ]; then
-    jdkw_arg=$(echo "${arg}" | grep '^JDKW_.*')
-    jdkw_base_dir_arg=$(echo "${arg}" | grep '^JDKW_BASE_DIR.*')
-    if [ -n "${jdkw_base_dir_arg}" ]; then
-      eval ${arg}
-    fi
-    if [ -n "${jdkw_arg}" ]; then
-      cmd_configuration="${cmd_configuration}${arg} "
-    fi
+  jdkw_arg=$(echo "${arg}" | grep '^JDKW_.*')
```

Comment:
What was this change? You no longer use the value of `in_command`?

I notice you do the same in https://github.com/ArpNetworking/metrics-aggregator-protocol/pull/10/files

Reply:
This is the bootstrap script for JDK Wrapper from here:

https://github.com/KoskiLabs/jdk-wrapper

This particular change is from:

https://github.com/KoskiLabs/jdk-wrapper/commit/772df10816ca353cf78b9515eb44c1e94328d0f7

The bootstrap script passes all the arguments to the actual JDK Wrapper script (`jdkw-impl.sh`) which is downloaded on-demand from the server.

The idea is that the bootstrap script should change less frequently than the actual wrapper script. The bootstrap script can also detect when it is out of date and tells the user after a run to update it. That's all I've done here and ideally should have been a separate PR (but if I don't do it when I see the message I'm likely to forget).